packages/webapp/pages/onboarding.tsx

```diff
@@ -186,21 +213,19 @@ export function OnboardPage(): ReactElement {
   };
 
   const onClickCreateFeed = () => {
-    const onboardingChecklist = getFeatureValue(feature.onboardingChecklist);
-
-    if (onboardingChecklist) {
-      setSettings({
-        sidebarExpanded: true,
-        onboardingChecklistView: ChecklistViewState.Open,
-      });
-    }
+    setSettings({
+      sidebarExpanded: true,
+      onboardingChecklistView: ChecklistViewState.Open,
+    });
 
     return onClickNext();
   };
 
-  const onSuccessfulLogin = () => {
+  const onSuccessfulLogin = useCallback(() => {
     router.replace(getPathnameWithQuery(webappUrl, window.location.search));
-  };
+    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
why did u add this? can't we add the dependencies?

Reply:
Router is not a stable reference 😭